pyproject.toml

```
@@ -19,6 +19,7 @@ Flask-JWT-Extended = "4.3.1"
 bcrypt = "3.2.2"
 authlib = "1.2.0"
 requests = "2.28.2"
+flask-paginate = "^2023.10.24"
```

Лучше версию зависимости зафиксировать. Автор может выпустить незначительное обновление, которое всё сломает, и при переустановке пакетов или при развёртывании системы на новом устройстве что-то пойдёт не так. Можно сделать так и перегенерировать `poetry.lock`:
```suggestion
flask-paginate = "2023.10.24"
```
По поводу мотивации полезно почитать вот эту статью об инцидентах в открытом программном обеспечении: https://maxleiter.com/blog/pin-dependencies